.github/workflows/build-container.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: Build Docker Container
+on:
+  push:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+
+      - name: Generate Docker Meta
+        uses: docker/metadata-action@v5
+        id: meta
+        with:
+          images: ghcr.io/kashalls/grasscutter
```

Comment:
reference to your own user?

Reply:
Replaced with `${{ github.repository }}` which returns [org_name/org_repo](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)